.github/workflows/test-suite.yml

```diff
@@ -154,7 +139,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
+      - uses: helix-editor/rust-toolchain@v1
         with:
           profile: minimal
           toolchain: 1.75.0
```

Comment:
I think we should probably remove the toolchain override here and just use the same as described in the toolchain.